backend/inspirehep/records/marshmallow/literature/bibtex.py

```diff
@@ -233,9 +234,17 @@ def get_title(self, data):
         title_parts = [title_dict["title"]]
         if "subtitle" in title_dict:
             title_parts.append(title_dict["subtitle"])
-        return ": ".join(
-            f"{{{latex_encode(part, contains_math=True)}}}" for part in title_parts
-        )
+
+        formatted_parts = []
+        for part in title_parts:
+            part_witouth_mathml = remove_tags(
+                remove_tags(part, allowed_tags="DUMMYROOTTAG")
+            )
+            latex_encoded_part = (
+                f"{{{latex_encode(part_witouth_mathml, contains_math=True)}}}"
+            )
```

Comment:
hmm I don't understand ` f"{{{ }}}"` need

Reply:
me neither, but it was like that before and I don't know latex and bibtex at all so I was afraid to remove it 😅 I can remove it

Reply:
Let’s try to remove it and see :)

Reply:
doesn't work, it's about adding `{}` to the encoded text, I'll leave it as it was